### PR TITLE
Consistent behaviour for document title/navigation/key

### LINKED ISSUE #8907: Consistent behaviour for document Title/Navigation/Key
The field order for "Add Page" is Title, Navigation, Key.
When changing the Title, the other fields get prefilled automatically.

image

The order in "Create Translation" (Document Language Overview) is Key, Navigation, Title.
Changing the Key prefills Navigation, Title stays empty. Changing the Title does nothing.

image

The same logic should be used for use cases.

In addition it would be nice to create a nice key/slug automatically.

As shown in the pimcore demo the following Keys are used:

image

Which results in URLs like: https://demo.pimcore.com/en/Magazine/Pick%20of%20the%20Day

In my opinion the Key should get prefilled with a nice slug (pick-of-the-day instead of Pick of the Day) if the field is empty.

Version: dev-master|add74462508e662a032f297149e1a7dfe5c042d2

### PR DESCRIPTION
**Pull request by valentin-**
## Changes in this pull request  
Resolves #8907

## Additional info  

- Changed field order
- Added/moved key event to title

### CODE REVIEW

**Comment by dvesh3** [comment]
@valentin- thanks for your PR. please sign CLA  https://github.com/pimcore/pimcore/pull/9753#issuecomment-879638066

File: bundles/AdminBundle/Resources/public/js/pimcore/document/document_language_overview.js [hunk 1]
```diff
@@ -242,15 +242,16 @@ pimcore.document.document_language_overview = Class.create({
                         },
                         items: [{
                             xtype: "textfield",
+                            itemId: "title",
```
Comment (dvesh3) [inline bundles/AdminBundle/Resources/public/js/pimcore/document/document_language_overview.js:245]: I would also make it consistent for translation new document option here https://github.com/pimcore/pimcore/blob/10.x/bundles/AdminBundle/Resources/public/js/pimcore/document/document.js#L481

Comment (valentin-) [reply bundles/AdminBundle/Resources/public/js/pimcore/document/document_language_overview.js:245]: @dvesh3 thanks, good input. I've added another commit for this.

**Comment by brusch** [comment]
Nice, thanks a lot! 👍

**Pull request merged**